pallets/randomness/Cargo.toml

```diff
@@ -19,34 +19,26 @@ sp-core = { workspace = true }
 sp-io = { workspace = true }
 sp-runtime = { workspace = true }
 sp-std = { workspace = true }
-sp-consensus-babe = { workspace = true }
 codec = { workspace = true }
 
-# Benchmarks
-frame-benchmarking = { workspace = true, optional = true }
```

Comment:
I agree with Alexandru, even though we have not benchmarked anything yet I'm not sure if it makes sense to remove a dependency that we know we'll need